app/renderer/js/pages/preference/new-server-form.ts

```diff
@@ -92,9 +92,8 @@ class NewServerForm extends BaseComponent {
 			this.submitFormHandler();
 		});
 		this.$newServerUrl.addEventListener('keypress', event => {
-			const EnterkeyCode = event.keyCode;
 			// Submit form when Enter key is pressed
-			if (EnterkeyCode === 13) {
```

Comment:
@SamuelMarks this looks good to me?

Reply:
See the deprecation notice https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
This was already fixed in 909e0f07e3f7439b0174b9a35bb72a1506bad0e2.